src/views/blog/data.json

```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "type": "Announcement",
+    "id": "contributeToScroll",
+    "date": "2023-07-13",
+    "title": "Announcing the Contribute to Scroll Initiative",
+    "posterTitle": "Contribute to Scroll",
+    "summary": "We are excited to announce our new Contribute to Scroll initiative.",
```

Comment:
consider using a more specific summary, this one here is a not-too-helpful repetition of the title itself